static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -78,10 +78,7 @@ const subscriptionUrl = addTrackingCodesToUrl({
 const signInUrl = `${signinHostname}/signin?utm_source=gdnwb&utm_medium=banner&utm_campaign=SubsBanner_Exisiting&CMP_TU=mrtn&CMP_BUNIT=subs`;
 
 const hasAcknowledged = bannerRedeploymentDate => {
-    // In order to migrate between ISO string and millisecond format support both temporarily
```

Comment:
@tomrf1 @jlieb10 @tjsilver I added this temporary workaround to the contributions banner as well in order to support both formats during a migration between the 2. You may find that your banner has been deployed in all regions by now, and therefore your timestamp format will be millisecond since epoch, and you can remove this.